src/events/event.cpp

```diff
@@ -205,6 +205,14 @@ void log(int i, const EventCameraPathFinished& e) {
     ));
 }
 
+void log(int i, const EventCameraMovedPosition& e) {
+    ghoul_assert(e.type == EventCameraMovedPosition::Type, "Wrong type");
+    LINFO(fmt::format(
+        "[{}] EventCameraMovedPosition",
+        i
+    ));
+}
```

Comment:
```suggestion
    LINFO(fmt::format("[{}] EventCameraMovedPosition", i));
}
```

Looks like it would fit in one line